package.json

```diff
@@ -24,6 +24,7 @@
   "author": "Mozilla (https://mozilla.org/)",
   "license": "MPL-2.0",
   "dependencies": {
+    "async": "2.0.0-rc.5",
```

Comment:
Da-Done! Once that gets merged, I will remove and re-wrap this and we should be good to go :)